modules/game/src/main/Game.scala

```diff
@@ -498,6 +498,39 @@ case class Game(
       p2Player = f(p2Player)
     )
 
+  // style "copy pasted" from a ts function
+  def multiPointResult(): String =
```

Comment:
you don't need the brackets if there are no inputs, this matches the style in lila.